README.md

```diff
@@ -34,51 +39,116 @@ We verify this package against N-2 recent versions of Go compiler. As of Septemb
 $ go get github.com/getsentry/sentry-go
 ```
 
-Or, if you are already using Go Modules, specify a version number as well:
+Or, if you are already using
+[Go Modules](https://github.com/golang/go/wiki/Modules), you may specify a
+version number as well:
 
 ```bash
-$ go get github.com/getsentry/sentry-go@v0.3.0
+$ go get github.com/getsentry/sentry-go@latest
 ```
 
+Check out the [list of released versions](https://pkg.go.dev/github.com/getsentry/sentry-go?tab=versions). 
+
 ## Configuration
 
-To use `sentry-go`, you’ll need to import the `sentry-go` package and initialize it with the client options that will include your DSN. If you specify the `SENTRY_DSN` environment variable, you can omit this value from options and it will be picked up automatically for you. The release and environment can also be specified in the environment variables `SENTRY_RELEASE` and `SENTRY_ENVIRONMENT` respectively.
```

Comment:
Reworked this section to have links to the various concepts/configs and tried to be more succinct.